package.json

```diff
@@ -375,6 +375,11 @@
           "type": "boolean",
           "default": true,
           "description": "Controls whether to use Utop for the REPL if it is installed in the current switch."
+        },
+        "ocaml.repl.evalTextBackgroundColor": {
+          "type": "string",
+          "format": "color-hex",
+          "description": "The background color of the text evaluated in REPL."
```

Comment:
Without a `default` value, the VS Code settings show an error: 
![image](https://github.com/ocamllabs/vscode-ocaml-platform/assets/25037249/3cd946a8-c597-4eb3-807f-727f5f225e4d)

Reply:
I updated its type as string or null.